packages/lesswrong/lib/types/libraryTypes.ts

```diff
@@ -3,11 +3,6 @@ declare module "deepmerge" {
   export default function deepmerge(a: any, b: any, options?: {isMergeableObject: any}): any
 }
 
-declare module "markdown-it-footnote" {
-  const markdownItFootnote: AnyBecauseTodo;
-  export default markdownItFootnote;
-}
```

Comment:
Looks like deleting this caused a type error in `markdownItPlugins`.  (Not sure what this change is motivated by)

Reply:
Adding `@types/markdown-it-footnote` made the assignments to things like `mdi.renderer.rules.footnote_block_open` typecheck. The typechecking error actually turned out to just be that `yarn.lock` was unpushed.